OpenSim/Simulation/Model/Muscle.h

```diff
@@ -441,6 +441,8 @@ OpenSim_DECLARE_ABSTRACT_OBJECT(Muscle, PathActuator);
     void extendSetPropertiesFromState(const SimTK::State &s) override;
     void extendInitStateFromProperties(SimTK::State& state) const override;
 
+    // Update the display geometry attached to the muscle
```

Comment:
Very sus

Reply:
This method exists in the current API (check the full diff). I added it back now that `updGeometryPath` is supported again.